commit 89dcd2d419755421290f85e32617acabdd81cac1
Author: lauren n. liberda <lauren@selfisekai.rocks>
Date:   Thu Jun 20 18:55:57 2024 +0000

    unbundle: add missing dav1d targets
    
    fixes "ERROR Unresolved dependencies.
    //third_party/crabbyavif:crabbyavif_dav1d_bindings(//build/toolchain/linux/unbundle:default)
      needs //third_party/dav1d:dav1d_config(//build/toolchain/linux/unbundle:default)
    //third_party/crabbyavif:crabbyavif_dav1d_bindings(//build/toolchain/linux/unbundle:default)
      needs //third_party/dav1d:dav1d_headers(//build/toolchain/linux/unbundle:default)"
    
    Change-Id: I85442e5fb67a804985354570fba453cc619c83d7
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5642761
    Reviewed-by: Lei Zhang <thestig@chromium.org>
    Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
    Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1317534}

diff --git a/build/linux/unbundle/dav1d.gn b/build/linux/unbundle/dav1d.gn
index 3d65158bc6144..a6d005a6622e3 100644
--- a/build/linux/unbundle/dav1d.gn
+++ b/build/linux/unbundle/dav1d.gn
@@ -21,3 +21,14 @@ source_set("dav1d") {
   deps = [ ":dav1d_shim" ]
   public_configs = [ ":system_dav1d" ]
 }
+
+group("dav1d_headers") {
+  public_deps = [ ":dav1d_shim" ]
+}
+
+config("dav1d_config") {
+  configs = [
+    ":system_dav1d",
+    ":dav1d_shim_config",
+  ]
+}
